client/components/domains/register-domain-step/index.jsx

```diff
@@ -353,6 +359,7 @@ const RegisterDomainStep = React.createClass( {
 							quantity: SUGGESTION_QUANTITY,
 							include_wordpressdotcom: this.props.includeWordPressDotCom,
 							include_dotblogsubdomain: this.props.includeDotBlogSubdomain,
+							tld_weight_overrides: this.getTldWeightOverrides(),
```

Comment:
The back end can accept a comma separated list for this value because I suspect that there may be other things that we do in the future to override TLD weights. Since I currently only return one item though it looks strange. But I wanted to set it up to allow this to return multiple values in the future.